plugin/kubernetes/README.md

```diff
@@ -65,6 +66,16 @@ kubernetes [ZONES...] {
      option requires substantially more memory than in insecure mode, since it will maintain a watch
      on all pods.
 
+* `endpoints` **ENDPOINT-MODE** sets the mode for handling service endpoint A
+   records, e.g.
+   `pod-name.my-service.svc.cluster.local. in A 1.2.3.4` if the pod is an
```

Comment:
Normally if the endpoint has a hostname then these will resolve. This change just makes them use the pod name if it exists and the hostname doesn't. I think that needs to be made clearer here.

Reply:
"If `hostname` is set on a pod, then it always has precedence." - is that enough?